app/crm/create-crm-case.js

```diff
@@ -0,0 +1,11 @@
+const publishCrmCase = require('../messaging/publish-crm-case')
+const { MessageSender } = require('ffc-messaging')
+const config = require('../config')
+
+const createCrmCase = async (crmCase) => {
+  const crmCaseSender = new MessageSender(config.createCrmCaseTopic)
+  publishCrmCase(crmCase, crmCaseSender)
+  console.info(`CRM Create Case Sent: ${crmCase}`)
```

Comment:
This will just log `[object Object]` as the CRM case.  Suggest just simplifying to create the agreement number or something for now until we know what the content of that message should be.